flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/NettyMessageClientDecoderDelegateTest.java

```diff
@@ -162,8 +162,9 @@ private void testNettyMessageClientDecoding(
 		try {
 			List<BufferResponse> messages = createMessageList(
 				hasEmptyBuffer,
-				hasBufferForReleasedChannel,
-				hasBufferForRemovedChannel);
+				hasBufferForRemovedChannel,
```

Comment:
These could just as well be naming errors in the method parameters.

Reply:
@gaoyunhaii Can you check if this change is valid?

Reply:
Yes, this is indeed not right and need fix. Very thanks for the fix! And if possible I think it might be better to swap the parameter orders of `createMessageList` instead to keep all the methods have the same parameter order.

Reply:
I changed the message signature.